catalogue_api/api/src/main/scala/uk/ac/wellcome/platform/api/controllers/WorksController.scala

```diff
@@ -93,26 +93,36 @@ class WorksController @Inject()(@Flag("api.prefix") apiPrefix: String,
 
     } { request: MultipleResultsRequest =>
       val pageSize = request.pageSize.getOrElse((defaultPageSize))
+      val includes = request.includes.getOrElse(WorksIncludes())
 
       val works = request.query match {
         case Some(queryString) =>
           worksService.searchWorks(
             queryString,
             pageSize = pageSize,
             pageNumber = request.page,
-            includes = request.includes.getOrElse(WorksIncludes()),
             index = request._index
           )
         case None =>
           worksService.listWorks(
             pageSize = pageSize,
             pageNumber = request.page,
-            includes = request.includes.getOrElse(WorksIncludes()),
             index = request._index
           )
       }
 
       works
+        .map { resultList =>
+          DisplayResultList(
```

Comment:
It might be worth moving some of the work done here to an apply method on `DisplayResultList` to make this less verbose.